kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/EntryClassUi.java

```diff
@@ -1035,9 +1035,34 @@ public void init() {
     }
 
     private void handleConcurrencyEvent(GwtEventInfo eventInfo) {
-        String infoMessage = (String) eventInfo.getProperties()
+        String eventOperation = (String) eventInfo.getProperties()
                 .get(ConfigurationChangeEvent.CONF_CHANGE_EVENT_INFO_PROP);
-        this.dropdownNotification.show(MSGS.configurationChangeNotification(infoMessage));
+        String eventPid = (String) eventInfo.getProperties().get(ConfigurationChangeEvent.CONF_CHANGE_EVENT_PID_PROP);
+
+        // unfourtunately, the enum in ConfigurationChangeEvent.Operations is not compiled here
```

Comment:
I believe we have internal APIs exported by the configuration service bundle. Maybe we could enrich them.